Update instructions for adding new data nodes (#582)
timescale/docs.timescale.com-content: getting-started/setup-multi-node-basic.md

forthcoming as TimescaleDB multi-node is made available on these cloud platforms.

## Initialize data nodes from the access node [](init_data_nodes_on_access_node)

Once logged in on the access node, it is necessary to add data nodes
to the local database before creating distributed hypertables. This
will make the data node available for use by distributed hypertables
and the access node will also connect to the data node and initialize
it.

While connected to the access node as a superuser (e.g., via `psql`),
use the command:

```sql
SELECT add_data_node('example_node_name', host => 'example_host_address');
```

`example_node_name` should be a unique name for the
node. `example_host_address` is the host name or IP address of the

> data node. It is also possible to specify a password to authenticate

Reviewer: ```suggestion
data node. You can specify a password to authenticate
```

with using the optional `password` parameter. But this is only
necessary if password authentication is used to connect to data nodes
and the password is not provided through other means (e.g., a local
password file). See the [`add_data_node`][add_data_node] API reference
documentation for detailed information about this command.

> It should now be possible to create distributed hypertables. Note,
> however, that in order to create and use distributed hypertables as a
> non-superuser, the user role needs to be distributed as well (i.e.,
> exist on all data nodes) and requires privileges to use the data nodes

Reviewer 2: Otherwise, it sounds more like you always need ot take this step, which you often might not need to do for testing.
```suggestion
It should now be possible to create distributed hypertables, e.g., 
running as the default `postgres` superuser.  If you wish to
create and use distributed hypertables as a
non-superuser, the user role needs to be distributed as well (i.e.,
```

> It should now be possible to create distributed hypertables. Note,
> however, that in order to create and use distributed hypertables as a
> non-superuser, the user role needs to be distributed as well (i.e.,
> exist on all data nodes) and requires privileges to use the data nodes
> on the access node. Please refer to the next section for instruction
> on how to create distributed roles and grant data node privileges.

Reviewer: Nit: the wording suggest that there is such a thing as a "distributed role" so I suggest to be more clear, also, suggest to remove the "should" and "possible" for a more straightforward wording.
```suggestion
You can now create distributed hypertables using `create_distributed_hypertable`, but note that in order to create and use distributed hypertables as a
non-superuser, the user role needs to exist on all data nodes and have the correct privileges to use the data nodes
on the access node. Please refer to the next section for instruction
on how to create roles on all data nodes and grant data node privileges.
```

Reviewer 2: Ha.  Mats already recommended.  Feel free to review both =)

Reviewer: I would probably make this even shorter and just say something like:
> the user role needs to exist on all data nodes and have the correct privileges on both the data and access node.

> ## (Optional) Add distributed user roles

Reviewer: The notion of a "distributed role" suggest that it is somehow maintained and is a novel concept. This section only talks about how to add roles on all nodes of a distributed database.
```suggestion
## (Optional) Adding roles for distributed hypertables
```

Author: "Distributed" is used as adjective here, and I think users know what it means for something to be "distributed".  I don't think it is a bad idea to introduce the concept of distributed roles or, more generally, distributed objects. Otherwise, it becomes very cumbersome to always say "roles that exist on all data nodes". 

I will try to be more "general" for now, but I think we need to figure out the proper terminology to use throughout the documentation.

Like any other database object, roles are _not_ distributed by

> default. In other words, a newly created role on the access node will

Reviewer: Hypertables and chunks are distributed, so this is not entirely true. Suggest to reformulate. (Just discussing roles is fine.)

Author: No, Hypertables and chunks are not distributed by default. You have to explicitly create a distributed hypertable, as opposed to a regular hypertable, for them to be distributed.

Reviewer: I think you should just remove the first sentence, it is redundant.

not exist on the data nodes and it will not be possible to use the
role to create and query distributed hypertables unless additional
steps are taken.

> To create a distributed role from the access node, use these commands:

Reviewer: ```suggestion
To create a role on all data nodes from the access node, use these commands:
```
